src/welcome.js

```diff
@@ -34,30 +34,18 @@ export default function Welcome({ application }) {
   const git_copy = builder.get_object("git_copy");
   git_copy.connect("clicked", () => {
     selectNone(git_text);
-    Clipboard.set_text(git_text.get_text(), -1);
-  });
-  git_text.connect("grab-focus", () => {
-    Promise.resolve().then(() => {
-      selectAll(git_text);
-    });
-    return false;
```

Comment:
ha yeah looks like navigating with `Tab` does this automatically on GTK4 - sweet

`selectAll` and `selectNone` can be removed and I found the issue with clipboard - here is a suggestion

```js
  const git_text = builder.get_object("git_text");
  git_text.label = `<tt>git config --global core.editor "${command}"</tt>`;
  const git_copy = builder.get_object("git_copy");
  git_copy.connect("clicked", () => {
    git_copy.get_clipboard().set(git_text.get_text());
  });

  const hg_text = builder.get_object("hg_text");
  hg_text.label = `<tt>[ui]\neditor=${command}</tt>`;
  const hg_copy = builder.get_object("hg_copy");
  hg_copy.connect("clicked", () => {
    hg_copy.get_clipboard().set(hg_text.get_text());
  });
```